src/syscall.h

```diff
@@ -8,14 +8,36 @@ See the file LICENSE for details.
 #define SYSCALL_H
 
 #include "kerneltypes.h"
+#include "graphics.h"
```

Comment:
I am going to move graphics_color into sys_window, and then have graphics include that. It seems the simplest solution here without exposing pure graphics functions to the users